docs/releasenotes.rst

```diff
@@ -1,6 +1,13 @@
 Release Notes
 =============
 
+R4-1: Changes since R4-0
+------------------------
+
+**This version has not yet been released**
```

Comment:
NB: Remove this comment before releasing.
Add other PR changes to these notes too...